ExternalModule.php

```diff
@@ -10,7 +10,9 @@ class ExternalModule extends AbstractExternalModule {
 
     function collectModules() {
         // fetch module id to name mappings
-        $sql = "SELECT * FROM redcap_external_modules;";
+        // Can't use WHERE `key` = 'enabled' and value = 'true' because $this->framework->query throws an exception. The probable cause is that `key` is not being escaped properly (the query works using MySQL workbench)
+        // Note: The results also contain disabled projects (projects disabled at the control center level and not at the project level)
+        $sql = "SELECT em.* FROM redcap_external_modules em INNER JOIN (select * from redcap_external_module_settings where value = 'true' and project_id is not null and type = 'boolean') settings ON em.external_module_id = settings.external_module_id;";
```

Comment:
```suggestion
        // Note: The results also contain disabled projects (projects disabled at the control center level and not at the project level)
        $sql = "SELECT DISTINCT em.* FROM redcap_external_modules em INNER JOIN
            (SELECT * FROM redcap_external_module_settings
                WHERE `key` = 'enabled'
                AND value = 'true'
                AND project_id IS NOT NULL  -- TODO: support system level config export in control center
            ) settings
            ON em.external_module_id = settings.external_module_id;";
```
Split up SQL over multiple lines, add a `DISTINCT` to limit the output (no functional change).  
I also was able to use "\`key\` = 'enabled'" with no issues so I swapped that in.